src/block_implementations/Eos/EosEncoderBlock.h

```diff
@@ -84,6 +88,9 @@ public slots:
 
     StringAttribute m_parameterName;
     BoolAttribute m_fineMode;
+    BoolAttribute m_active;
+    BoolAttribute m_accelerate;
+    BoolAttribute m_feedback;
```

Comment:
Maybe rename to m_feedbackEnabled (as well as the attribute name in the constructor) to be consistent with the Fader Bank block.